decoupled-site/frontend/src/graphql/Search.graphql

```diff
@@ -0,0 +1,23 @@
+query Search($searchParam: String, $locales: Locales, $order: OrderBy) {
+  ArtistDetailsPage(
+    locale: [$locales]
+    orderBy: { ArtistName: $order }
+    where: { _or: [{ Name: { contains: $searchParam, boost: 10 } }, { Name: { startsWith: $searchParam, boost: 10 } }, { StageName: { startsWith: $searchParam } }] }
+  ) {
+    items {
+      PerformanceStartTime
+      PerformanceEndTime
+      StageName
+      ArtistName
+      ArtistPhoto
```

Comment:
Why are we not displaying the values that you have selected in the result list? Instead of a snippet (which are not supporting yet), could you just display the whole discription and include it as XHTML?

![Screenshot 2023-04-24 at 12 40 26](https://user-images.githubusercontent.com/1325546/233974069-edb16bff-d9da-40a9-96a1-347561cbbcd7.png)